manifests/claudie/frontend.yaml

```diff
@@ -21,14 +21,14 @@ spec:
       serviceAccountName: frontend-sa
       containers:
         - name: frontend
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
Why was this changed ?

Reply:
Sorry 🙏 , I forgot to change this back when developing locally - will change this back